packages/snaps-execution-environments/scripts/start.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+const http = require('http');
+const path = require('path');
+const serveHandler = require('serve-handler');
+
+const ROOT = path.resolve(__dirname, '..');
+const PUBLIC = path.join(ROOT, '/dist/browserify/iframe');
+
+start();
+
+/**
+ * Script to serve a local instance of the Iframe Execution Environment.
+ * Expects that the first positional argument to the Node process is a valid
+ * integer port.
+ */
+async function start() {
+  const port = parseInt(process.argv[2] ?? 8000, 10);
```

Comment:
```suggestion
  const port = parseInt(process.argv[2] ?? 6363, 10);
```
Can we use something that doesn't overlap with the ports we use for snaps?